Source/Core/Common/CRC32.h

```diff
@@ -0,0 +1,9 @@
+// Copyright 2021 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include <string>
+
+#include "Common/CommonTypes.h"
+
+u32 ComputeCRC32(const std::string& data);
```

Comment:
`std::string_view` would be preferable here.

Reply:
Thanks, fixed!